packages/lesswrong/components/posts/TruncatedAuthorsList.tsx

```diff
@@ -28,33 +28,52 @@ const styles = (_: ThemeType) => ({
   },
 });
 
-const reformatAuthorPlaceholder = (
-  moreCount: number,
-  totalItems: number,
-  moreNode: Element,
-) => {
-  const text = moreCount === totalItems
-    ? `${moreCount} authors`
-    : `+ ${moreCount} more`
-  moreNode.innerHTML = moreNode.innerHTML.replace(
-    /(\+ \d+ more)|(\d+ authors)/,
-    text,
-  );
-}
-
-const TruncatedAuthorsList = ({post, expandContainer, className, classes}: {
+const TruncatedAuthorsList = ({
+  post,
+  expandContainer,
+  className,
+  classes,
+  useMoreSuffix = true
+}: {
   post: PostsList | SunshinePostsList | PostsBestOfList,
   expandContainer: RefObject<HTMLDivElement>,
   className?: string,
   classes: ClassesType<typeof styles>,
+  useMoreSuffix?: boolean,
 }) => {
   const {isAnon, authors, topCommentAuthor} = usePostsUserAndCoauthors(post);
   const ref = useRef<HTMLDivElement>(null);
+  const [hiddenAuthors, setHiddenAuthors] = useState<UsersMinimumInfo[]>([]);
 
   useEffect(() => {
     if (isAnon || authors.length === 0) {
+      setHiddenAuthors([]);
       return;
     }
+    const reformatAuthorPlaceholder = (
```

Comment:
1. there's no reason to move this function definition into the `useEffect`; if you want to call `setHiddenAuthors` on the hidden authors you can change the function to return them and call `setHiddenAuthors` on the return value.
2. this component is used all over the place and I can't tell if these changes are a no-op by default

Reply:
it wasn't a nullop by default (changed with authors were listed in the tooltip for all uses) but I've reverted that because keeping it'd be more work than I feel like doing rn